boot/modeenv.go

```diff
@@ -123,6 +124,14 @@ func (m *Modeenv) Write() error {
 	return fmt.Errorf("internal error: must use WriteTo with modeenv not read from disk")
 }
 
+// Origin returns where the modeenv was read from if it was read.
+func (m *Modeenv) Origin() (string, error) {
```

Comment:
not sure we need this, isn't what Write will do for us?

Reply:
removed